src/scss/grommet-core/_objects.table.scss

```diff
@@ -88,4 +122,12 @@
       }
     }
   }
+
+  @include media-query(palm) {
```

Comment:
Are we sure that we would never want to display a table in the typical way on a palm sized device? I could see a two column table working just fine. Can we just tie this to the .table--small selector?